shared/js/smart-screen/card_manager.js

```diff
@@ -671,6 +671,13 @@
     findCardFromCardList: function cm_findCardFromCardList(query) {
       var found;
       this._cardList.some(function(card) {
+        if (card instanceof Folder) {
+          found = card.findCard(query);
```

Comment:
Filed [bug 1156726](https://bugzilla.mozilla.org/show_bug.cgi?id=1156726) for it